### PR TITLE
Ignore flaky UI Test, for now (see #1482, fixes #2197)

### LINKED ISSUE #2197: CI is broken and fails for trivial PR
**Describe the bug**
My trivial PR #2196, which only adds a `.github/dependabot.yaml` file, which IMHO is "impossible" to affect test runs and such things, failed to pass CI and is red instead of green.

This can discourage contributions (because you never clearly know if the change proposed in a PR breaks CI, or if it was "just already broken anyway"), and therefore perhaps should be fixed with top priority, over everything else?

IMHO any project should intentionally set itself a relatively high bar in this matter, and aim for CI on its main branch to always be green. Any PRs that break it and turn it red should be immediately be reverted by maintainers.

Glancing over https://github.com/google/android-fhir/commits/master this historically does not seem to be the case in this project.

**To Reproduce**
1. Raise a trivial PR e.g. adding a space to the README

**Expected behavior**
It can never say _Some checks were not successful_ for such trivial PR changes.

**Would you like to work on the issue?**
I am happy to help "drive" this, but suspect it will need input from more people.

/Cc @fredhersch @jingtang10 @omarismail94 @williamito FYA

### PR DESCRIPTION
**Pull request by vorburger**
Fixes #2197, see also #1482

**Description**
This disables a known flaky UI test. This reduces confusion on why PRs fail to build.

**Alternative(s) considered**
_Have you considered any alternatives? And if so, why have you chosen the approach in this PR?_
The alternative is to fix the flaky UI test. My Android foo isn't strong enough to know how to do that, unfortunately.

**Type**
Choose one: **Bug fix** ~Feature | Documentation | Testing | Code health | Builds | Releases | Other~

### CODE REVIEW

File: datacapture/src/androidTest/java/com/google/android/fhir/datacapture/test/views/QuestionnaireItemDialogMultiSelectViewHolderFactoryEspressoTest.kt [hunk 1]
```diff
@@ -261,6 +262,7 @@ class QuestionnaireItemDialogMultiSelectViewHolderFactoryEspressoTest {
   }
 
   @Test
+  @Ignore // TODO https://github.com/google/android-fhir/issues/1482 FIXME
```
Comment (omarismail94) [inline datacapture/src/androidTest/java/com/google/android/fhir/datacapture/test/views/QuestionnaireItemDialogMultiSelectViewHolderFactoryEspressoTest.kt:265]: @Ignore takes a parameter where you can add the link:
```suggestion
  @Ignore("https://github.com/google/android-fhir/issues/1482")
```

**Review by jingtang10 — CHANGES_REQUESTED** [review]
sorry for the delay... i thought i sent this out: but my main comment here is can you try to ignore this only for api level 24? this is not flaky on api level 33.

**Comment by vorburger** [comment]
> sorry for the delay... i thought i sent this out: but my main comment here is can you try to ignore this only for api level 24? this is not flaky on api level 33.

I have no idea if it's possible to (easily) do that. @omarismail94 would you happen to know?

If it's not, my recommendation is to ignore it for all, until fixed, to have green CI.

**Comment by omarismail94** [comment]
> > sorry for the delay... i thought i sent this out: but my main comment here is can you try to ignore this only for api level 24? this is not flaky on api level 33.
> 
> I have no idea if it's possible to (easily) do that. @omarismail94 would you happen to know?
> 
> If it's not, my recommendation is to ignore it for all, until fixed, to have green CI.

In the flaky test, you can add this block:

```
    @Test
    fun testMethodToIgnoreOnApi24() {
        // Ignore this specific test if the API level is 24
        Assume.assumeTrue(Build.VERSION.SDK_INT != Build.VERSION_CODES.N)
        
        // Your test code here...
    }
```

https://github.com/google/android-fhir/pull/2203#issuecomment-1738892206

**Comment by vorburger** [comment]
An alternative suggested by @jingtang10 is https://developer.android.com/reference/androidx/test/filters/SdkSuppress.

**Review by omarismail94 — COMMENTED** [review]
@vorburger can you also add the same thing to:
`clickAddAnotherAnswer_shouldScrollDownToShowAddAnotherAnswer`

It is the other flaky test this library

**Review by omarismail94 — APPROVED** [review]

**Review by jingtang10 — APPROVED** [review]

**Pull request merged**